From cf218d82862e986e544bf43606068767e6cb8eaa Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:10:57 -0400
Subject: [usb] pl2303: use C_CMSPAR macro

Message-id: <1410887463-20674-586-git-send-email-dzickus@redhat.com>
Patchwork-id: 95484
O-Subject: [RHEL7 PATCH 585/591] USB: pl2303: use C_CMSPAR macro
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 619c43543a946dd2f094d2f1bfc81a3fd48356d8
Author: Johan Hovold <jhovold@gmail.com>
Date:   Sun Dec 29 19:23:18 2013 +0100

    USB: pl2303: use C_CMSPAR macro

    Use the new C_CMSPAR macro for consistency.

    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/pl2303.c b/drivers/usb/serial/pl2303.c
index 8707d0d..6ec9f3c 100644
--- a/drivers/usb/serial/pl2303.c
+++ b/drivers/usb/serial/pl2303.c
@@ -527,7 +527,7 @@ static void pl2303_set_termios(struct tty_struct *tty,
   /* For reference buf[5]=3 is mark parity */
   /* For reference buf[5]=4 is space parity */
   if (C_PARODD(tty)) {
-   if (tty->termios.c_cflag & CMSPAR) {
+   if (C_CMSPAR(tty)) {
     buf[5] = 3;
     dev_dbg(&port->dev, "parity = mark\n");
    } else {
@@ -535,7 +535,7 @@ static void pl2303_set_termios(struct tty_struct *tty,
     dev_dbg(&port->dev, "parity = odd\n");
    }
   } else {
-   if (tty->termios.c_cflag & CMSPAR) {
+   if (C_CMSPAR(tty)) {
     buf[5] = 4;
     dev_dbg(&port->dev, "parity = space\n");
    } else {
-- 
1.7.1